lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -104,7 +104,7 @@ SmallVector<unsigned, 2> warpsPerTileMI200(triton::DotOp dotOp,
     if (ret[0] * ret[1] >= numWarps)
       break;
     if (tensorShape[0] / shapePerWarp[0] / ret[0] >=
```

Comment:
```suggestion
    if (tensorShape[0] / (shapePerWarp[0] * 2) / ret[0] >=
```

This enables even bigger matrices, need to tune matrix multiplication tutorial to improve it's performance